src/Movie.java

```diff
@@ -3,8 +3,28 @@ public class Movie {
     String genre;
     int releaseYear;
     boolean includedInPlan;
-    double averageRate;
+    double rate;
     double ratingCounts;
     double durationInMinutes;
 
+    void avaliation(double avaliation) {
+        rate += avaliation;
+        ratingCounts++;
+    }
+
+    double media() {
+        if (ratingCounts > 0) {
+            return rate / ratingCounts;
+        } else {
+            return 0;
```

Comment:
As we talked previously, you could return something like -1 or null instead of 0, this would allow you to differenciate when you have no ratings from when the average is 0